demo/compose-s3-aws.yml

```diff
@@ -33,16 +46,20 @@ services:
       # Enable Tiered Storage
       KAFKA_REMOTE_LOG_STORAGE_SYSTEM_ENABLE: true
       KAFKA_REMOTE_LOG_MANAGER_TASK_INTERVAL_MS: 5000
-      # Tiered Storage Remote Metadata Manager
+      # Remote metadata manager
       KAFKA_REMOTE_LOG_METADATA_MANAGER_CLASS_NAME: "org.apache.kafka.server.log.remote.metadata.storage.TopicBasedRemoteLogMetadataManager"
       KAFKA_RLMM_CONFIG_REMOTE_LOG_METADATA_TOPIC_REPLICATION_FACTOR: 1
       KAFKA_REMOTE_LOG_METADATA_MANAGER_LISTENER_NAME: BROKER
-      # Tiered Storage Remote Storage Manager
-      KAFKA_REMOTE_LOG_STORAGE_MANAGER_CLASS_NAME: "io.aiven.kafka.tieredstorage.RemoteStorageManager"
+      # Remote storage manager
       KAFKA_REMOTE_LOG_STORAGE_MANAGER_CLASS_PATH: "/tiered-storage-for-apache-kafka/*"
+      KAFKA_REMOTE_LOG_STORAGE_MANAGER_CLASS_NAME: "io.aiven.kafka.tieredstorage.RemoteStorageManager"
```

Comment:
Should we keep the terminology aligned with Kafka, e.g. instead of KAFKA_REMOTE_LOG_STORAGE_MANAGER_CLASS_NAME use KAFKA_REMOTE_STORAGE_MANAGER_CLASS_NAME?

Reply:
`KAFKA_REMOTE_LOG_STORAGE_MANAGER_CLASS_NAME` is a Kafka's setting, we can't control it